### ISSUE TITLE
Android - Steps Screen - Incorrect Heading Level

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#29]
Why is the “Step” heading set at a heading level three (h3) when there are no other headings on the screen I would expect it to be a heading level one (h1) since it’s the first one and any others should be a heading level two (because they fall under the main one.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

Point of clarification for me (and forgive my ignorance), but why does the heading level need to start at 1 at the top? As you likely saw, the heading level 1 has really large font. Is it a 508 compliance issue if we start at heading level 2 or 3? As long as we go sequentially down, (i.e. main heading level is 2, then subsequent levels are 3, 4, etc.), is that okay?

Thanks,
Kyle

**Comment by jfc3-dol** [comment]
The best way I have to describe the heading structure is the following.

You only have one book TITLE so that would be a heading level one (h1). Each chapter would be a heading level two (h2). A section in the chapter would be a heading level three (h3), and sub-section would be a heading level four (h4). You're not going to skip a level. We are talking about paper books here.

Or as a buddy of mine explains it the heading level one (h1) is the name restaurant and the main types of food are the heading level two (h2) (soup, salad, appetizers, mains, etc. The names of the dishes or categories of dishes would be the heading level three (h3) items such as chicken, fish/seafood, pork, beef, etc.

**Comment by KyleLivingstonDOL** [comment]
Thanks John. However, it would not be wrong to start at Heading level 2 or 3 if that is the initial heading level, correct? I.e. the talk back will not read "Heading Level 1, Step 1 XXXXXXX" correct? That way, since H1 is really large and takes up a lot of space on a mobile device, we could start with H3 or H4 and work our way down from there.

Thanks,
Kyle

**Comment by jfc3-dol** [comment]
If the code is written in HTML be that generated from JavaScript or plain HTML TalkBack and other screen readers will read "Heading level one Step 1 XYZ".

If CSS is being used, you can then at certain widths make the heading level one (h1) text larger or smaller. Doing so will allow the heading level one (h1) text to be huge on a narrow width like a cell phone.

**Comment by KyleLivingstonDOL** [comment]
Hi John,

We've found a workaround to this. It will be rolled out for the next UAT on June 10. Thanks for your insight into this.

-Kyle

**Comment by jfc3-dol** [comment]
Closing issue because TalkBack is now reading the Step heading as a heading level one (h1).
